app/contributing/word-choice.md

```diff
@@ -60,13 +61,29 @@ Use <i class="fa fa-check"></i> | Don't use <i class="fa fa-times"></i>
 Self-managed                    | Self-hosted, on-premise(s)
 Free tier                       | Free mode
 Plus tier                       | Plus mode, Plus subscription
-Enteprise tier                  | Enterprise mode, Enterprise subscription
+Enterprise tier                  | Enterprise mode, Enterprise subscription
 Cloud                           | Software as a Service/SaaS
 cURL                            | curl, CURL
 HTTPie                          | httpie, HTTPIE
 Unix                            | unix, UNIX
 Nginx                           | nginx, NGINX
 
+
+### List punctuation
+Don't use punctuation when creating ordered and unordered lists that are extensions of a sentence:
+Kong Mesh enables the microservices transformation with:
+- Out-of-the-box service connectivity and discovery
+- Zero-trust security
+- Traffic reliability
+- Global observability across all traffic
```

Comment:
Maybe stick this into a codeblock to make it stand out as an example? In the very least, it needs another line break before the "Kong Mesh" sentence.